src/supporting-data/keywords/components/entry/styles/keywords-graph.module.scss

```diff
@@ -0,0 +1,32 @@
+.graph {
+  position: relative;
+  width: 500px;
+  height: calc(50vh - 100px);
```

Comment:
What do you think about making the height of the diagram dependent on the number of rows of ancestors? Eg when there are only two nodes of a parent and child the diagram will be shorter than a diagram with three nodes of a grandparent, parent and child?